Refactornew

bubonicfred opened the pull request.

bubonicfred committed:
183 remove babel plugin add module exports (#189)

* Update package.json

* Update .babelrc

bubonicfred committed:
Update package.json

Update wdio
Fix chromedriver ver

bubonicfred committed:
Merge branch 'buildfix' into buildfix-babel

bubonicfred committed:
WDIO migration

bubonicfred committed:
Update wdio.conf.js

bubonicfred committed:
Update package.json

Add mocha-framework

bubonicfred committed:
Update package.json

update babel runtime

bubonicfred committed:
Update package.json

Add missing babel plugin

bubonicfred committed:
bson

Replace mongodb EJSON with bson

bubonicfred committed:
Update package.json

Bump babel versions

bubonicfred committed:
Update versions

bubonicfred committed:
Update package.json

update mocha

bubonicfred committed:
Update wdio.conf.js

Comment out compilers

bubonicfred committed:
Update package.json

Redirect babel references to new location

bubonicfred committed:
Merge pull request #388 from bubonicfred/buildfix-babel

Various changes for updating babel

bubonicfred committed:
add webpack

bubonicfred committed:
Update package.json

remove inject-loader
not used

bubonicfred committed:
Update package.json

Remove some obsolete packages

bubonicfred committed:
Buildfix babel (#421)

* Add wdio mocha framework

* replace mocha.opts

* Config updates

* various package updates

* Config + package updates

* Replace deprecated sinon call

* More deprecated call replacement

* add node 14 test,

* build: :heavy_minus_sign:  @wdio/sync deprecated


* chore: :arrow_up: Upgrade meteor to 2.13

* chore: :see_no_evil: Add generated config file to .gitignore

* Restyled by clang-format

* Restyled by jq

* Restyled by prettier

* Restyled by prettier-json

* chore: :pushpin: Pin babel minor versions

---------

Co-authored-by: restyled-io[bot] <32688539+restyled-io[bot]@users.noreply.github.com>
Co-authored-by: Restyled.io <commits@restyled.io>

deepsource-io[bot] committed:
ci: add .deepsource.toml

depfu[bot] committed:
chore(package): update sinon to version 16.0.0

depfu[bot] committed:
chore(package): update sinon to version 16.0.0

depfu[bot] committed:
chore(package): update bson to version 6.1.0 (#437)

Co-authored-by: depfu[bot] <23717796+depfu[bot]@users.noreply.github.com>

depfu[bot] committed:
chore(package): update bson to version 6.1.0 (#437)

Co-authored-by: depfu[bot] <23717796+depfu[bot]@users.noreply.github.com>

depfu[bot] committed:
Pin randomstring to version 1.3.0 (#427)

Co-authored-by: depfu[bot] <23717796+depfu[bot]@users.noreply.github.com>

depfu[bot] committed:
Pin randomstring to version 1.3.0 (#427)

Co-authored-by: depfu[bot] <23717796+depfu[bot]@users.noreply.github.com>

depfu[bot] committed:
chore(package): update node-sass to version 9.0.0

depfu[bot] committed:
chore(package): update md5-file to version 5.0.0

depfu[bot] committed:
chore(package): update md5-file to version 5.0.0

depfu[bot] committed:
chore(package): update semver to version 7.5.4

depfu[bot] committed:
chore(package): update semver to version 7.5.4

bubonicfred committed:
chore: :heavy_minus_sign: Prune unused packages from pacakge.json

bubonicfred committed:
chore: :heavy_minus_sign: Prune unused packages from pacakge.json

bubonicfred committed:
refactor:

bubonicfred committed:
refactor: :alien: Update code to replace depecated faker call (for v6/7)

bubonicfred committed:
chore(package): :arrow_up: Update faker to v6

depfu[bot] committed:
chore(package): update node-getopt to version 0.3.2

depfu[bot] committed:
chore(package): update node-getopt to version 0.3.2

bubonicfred committed:
refactor: refactor randomstringcode

bubonicfred committed:
style: style class

deepsource-autofix[bot] committed:
refactor: replace builtin prototype method with `Object.prototype` equivalent

It is preferable to call certain `Object.prototype` methods through `Object` on object instances instead of using the builtins directly.

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 05a566f according to the output
from Prettier.

Details: https://github.com/bubonicfred/5minitz/pull/451

bubonicfred committed:
chore(package): :arrow_down: rollback randomstring version

deepsource-autofix[bot] committed:
refactor: replaces `Meteor.setTimeout` with `Meteor.defer`

`Meteor.setTimeout` can be used to defer the execution of a function, but Meteor has a built-in method for deferring called `Meteor.defer`. It is better to use the dedicated method instead of relying on a side-effect of `Meteor.setTimeout`.
Using `Meteor.defer` is preferred because it uses native `setImmediate` or `postMessage` methods if available. Otherwise, it can fall back to `setTimeout`. Also, `setTimeout` adds a delay of at least 2ms in Chrome, 10ms in other browsers, which results in performance issues.

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in a60210a according to the output
from Prettier.

Details: https://github.com/bubonicfred/5minitz/pull/452

bubonicfred committed:
refactor:

bubonicfred committed:
refactor: :recycle: reuse random class

bubonicfred committed:
fix: :alien: Update code for Faker v8

bubonicfred committed:
fix: :recycle: Correct path to module

bubonicfred committed:
refactor: :alien: Replace deprecated faker call

bubonicfred committed:
refactor: :alien: Update deprecated faker calls

bubonicfred committed:
fix: :lock: narrow regex scope

bubonicfred committed:
chore(package): Move faker to runtime deps to test

bubonicfred committed:
fix: :lock: same as before

bubonicfred committed:
refactor:

restyled-commits committed:
Restyled by clang-format

restyled-commits committed:
Restyled by prettier

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 3d9a22d according to the output
from Prettier.

Details: https://github.com/bubonicfred/5minitz/pull/456

restyled-commits committed:
Restyled by clang-format

restyled-commits committed:
Restyled by prettier

deepsource-autofix[bot] committed:
refactor: convert logical operator to optional chainining

The [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) operator can be used to perform null checks before accessing a property, or calling a function.

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in cf99118 according to the output
from Prettier.

Details: https://github.com/bubonicfred/5minitz/pull/460

bubonicfred committed:
Merge branch 'develop_new' into buildfix

restyled-commits committed:
Restyled by clang-format

restyled-commits committed:
Restyled by prettier

depfu[bot] committed:
chore(package): update chromedriver to version 117.0.3

bubonicfred committed:
Merge changes in from buidfix branch (#461)

* chore(package): update bson to version 6.1.0 (#437)

Co-authored-by: depfu[bot] <23717796+depfu[bot]@users.noreply.github.com>

* Pin randomstring to version 1.3.0 (#427)

Co-authored-by: depfu[bot] <23717796+depfu[bot]@users.noreply.github.com>

* chore(package): update sinon to version 16.0.0

* chore(package): update md5-file to version 5.0.0

* chore(package): update semver to version 7.5.4

* chore: :heavy_minus_sign: Prune unused packages from pacakge.json

* chore(package): update node-sass to version 9.0.0

* refactor:

* refactor: :alien: Update code to replace depecated faker call (for v6/7)

* chore(package): :arrow_up: Update faker to v6

* refactor: refactor randomstringcode

* style: style class

* chore(package): :arrow_down: rollback randomstring version

* refactor:

* refactor: :recycle: reuse random class

* fix: :alien: Update code for Faker v8

* fix: :recycle: Correct path to module

* refactor: :alien: Replace deprecated faker call

* refactor: :alien: Update deprecated faker calls

* chore(package): Move faker to runtime deps to test

* refactor:

* Restyled by clang-format

* Restyled by prettier

* Restyled by clang-format

* Restyled by prettier

---------

Co-authored-by: depfu[bot] <23717796+depfu[bot]@users.noreply.github.com>
Co-authored-by: Restyled.io <commits@restyled.io>

depfu[bot] committed:
chore(package): update markdownlint-cli to version 0.37.0 (#464)

Co-authored-by: depfu[bot] <23717796+depfu[bot]@users.noreply.github.com>

depfu[bot] committed:
chore(package): update ddp to version 0.12.1 (#441)

Co-authored-by: depfu[bot] <23717796+depfu[bot]@users.noreply.github.com>

deepsource-autofix[bot] committed:
refactor: fix Yoda conditions (#465)

* refactor: fix Yoda conditions

Yoda conditions are named so because the literal value of the condition comes first while the variable comes second.

For instance,

```js
if ("red" === color) {
    // ...
}
```

Yoda condition is fixed by switching the literal and variable.

* style: format code with Prettier

This commit fixes the style issues introduced in 3c46593 according to the output
from Prettier.

Details: https://github.com/bubonicfred/5minitz/pull/465

---------

Co-authored-by: deepsource-autofix[bot] <62050782+deepsource-autofix[bot]@users.noreply.github.com>

bubonicfred committed:
build: :hammer: Format and lint test script (#466)

* build: :hammer: Format and lint test script

* Restyle build: :hammer: Format and lint test script (#467)

* Restyled by shellharden

* Restyled by shfmt

---------

Co-authored-by: Restyled.io <commits@restyled.io>

---------

Co-authored-by: restyled-io[bot] <32688539+restyled-io[bot]@users.noreply.github.com>
Co-authored-by: Restyled.io <commits@restyled.io>

bubonicfred committed:
Unpin Babel to 7.22 (#449)

bubonicfred committed:
refactor: :coffin: Remove references to cloudprint as deprecated (#468)

* refactor: :coffin: Remove references to cloudprint as deprecated

* style: format code with Prettier

This commit fixes the style issues introduced in 1f9a632 according to the output
from Prettier.

Details: https://github.com/bubonicfred/5minitz/pull/468

* Restyle refactor: :coffin: Remove references to cloudprint as deprecated (#469)

* Restyled by clang-format

* Restyled by prettier

---------

Co-authored-by: Restyled.io <commits@restyled.io>

---------

Co-authored-by: deepsource-autofix[bot] <62050782+deepsource-autofix[bot]@users.noreply.github.com>
Co-authored-by: restyled-io[bot] <32688539+restyled-io[bot]@users.noreply.github.com>
Co-authored-by: Restyled.io <commits@restyled.io>

bubonicfred committed:
chore(package): :heavy_minus_sign: Remove babel polyfill (#471)

bubonicfred committed:
perf: remove redundant await calls (#472)

* perf: remove redundant await calls

* style: format code with Prettier

This commit fixes the style issues introduced in 56e615c according to the output
from Prettier.

Details: https://github.com/bubonicfred/5minitz/pull/472

* Restyle perf: remove redundant await calls (#473)

* Restyled by clang-format

* Restyled by prettier

---------

Co-authored-by: Restyled.io <commits@restyled.io>

---------

Co-authored-by: deepsource-autofix[bot] <62050782+deepsource-autofix[bot]@users.noreply.github.com>
Co-authored-by: restyled-io[bot] <32688539+restyled-io[bot]@users.noreply.github.com>
Co-authored-by: Restyled.io <commits@restyled.io>

deepsource-autofix[bot] committed:
refactor: remove trailing `undefined` argument from function call (#474)

* refactor: remove trailing `undefined` argument from function call

When an argument is omitted from a function call, it will default to `undefined`. It is therefore redundant to explicitly pass an `undefined` literal as the last argument.

* style: format code with Prettier

This commit fixes the style issues introduced in ea27a47 according to the output
from Prettier.

Details: https://github.com/bubonicfred/5minitz/pull/474

---------

Co-authored-by: deepsource-autofix[bot] <62050782+deepsource-autofix[bot]@users.noreply.github.com>

bubonicfred committed:
chore(package): :heavy_plus_sign: Change ddp client,

bubonicfred committed:
chore(package): :heavy_plus_sign: Change ddp client,

bubonicfred committed:
chore: :heavy_plus_sign: add ddp client

bubonicfred committed:
chore: :heavy_plus_sign: add ddp client

bubonicfred committed:
Buildfix apicheck (#475)

* test: :alien: Add flag to check for old api usage in meteor

* test:

* test: :technologist: Add trace warnings flag to test run

* test: :bug: fix shell script

* Restyled by shfmt (#476)

Co-authored-by: Restyled.io <commits@restyled.io>

---------

Co-authored-by: restyled-io[bot] <32688539+restyled-io[bot]@users.noreply.github.com>
Co-authored-by: Restyled.io <commits@restyled.io>

bubonicfred committed:
build: :rotating_light: lint shell scripts (#477)

* build: :rotating_light: lint shell scripts

* Restyle build: :rotating_light: lint shell scripts (#478)

* Restyled by shellcheck

* Restyled by shellharden

* Restyled by shfmt

---------

Co-authored-by: Restyled.io <commits@restyled.io>

---------

Co-authored-by: restyled-io[bot] <32688539+restyled-io[bot]@users.noreply.github.com>
Co-authored-by: Restyled.io <commits@restyled.io>

bubonicfred committed:
Merge branch 'develop_new' into buildfix

deepsource-autofix[bot] committed:
refactor: replace short hand type conversions with function calls

Prefer using explicit casts by calling `Number`, `Boolean`, or `String` over using operators like `+`, `!!` or `"" +`. This is considered best practice as it improves readability.

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 68653d0 according to the output
from Prettier.

Details: https://github.com/bubonicfred/5minitz/pull/480

bubonicfred committed:
Merge pull request #479 from bubonicfred/buildfix

Merge in Maintenance from buildfix

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 439b0c4 according to the output
from Prettier.

Details: https://github.com/bubonicfred/5minitz/pull/481

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 439b0c4 according to the output
from Prettier.

Details: https://github.com/bubonicfred/5minitz/pull/481

bubonicfred committed:
chore(meteor): :arrow_up: bump meteor to 2.13.3

bubonicfred committed:
chore(meteor): :arrow_up: bump meteor to 2.13.3

bubonicfred committed:
chore(meteor): :arrow_up: upgrade alanning:roles

bubonicfred committed:
chore(meteor): :arrow_up: upgrade alanning:roles

bubonicfred committed:
chore(meteor): :heavy_minus_sign: Remove markdown package (deprecated)

bubonicfred committed:
chore(meteor): :heavy_minus_sign: Remove markdown package (deprecated)

bubonicfred committed:
chore(meteor): :arrow_up: Upgrade ostrio:files

bubonicfred committed:
chore(meteor): :arrow_up: Upgrade ostrio:files

bubonicfred committed:
build: :rotating_light: harden launch script

bubonicfred committed:
build: :rotating_light: harden launch script

deepsource-autofix[bot] committed:
refactor: replace `parseInt()` with non-decimal literal

The `parseInt()` and `Number.parseInt()` functions can be used to turn binary, octal, and hexadecimal strings into integers. As binary, octal, and hexadecimal literals are supported in ES6, we recommend the use of those numeric literals instead of `parseInt()` or `Number.parseInt()`.

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in d9ab888 according to the output
from Prettier.

Details: https://github.com/bubonicfred/5minitz/pull/485

restyled-commits committed:
Restyled by shellharden

restyled-commits committed:
Restyled by shellharden

restyled-commits committed:
Restyled by shfmt

restyled-commits committed:
Restyled by shfmt

bubonicfred committed:
chore(meteor): :arrow_up: Bump msavin:mongol

bubonicfred committed:
chore(meteor): :arrow_up: Bump msavin:mongol

bubonicfred committed:
chore(package): :heavy_minus_sign: Remove xtend as unused

bubonicfred committed:
chore(package): :heavy_minus_sign: Remove xtend as unused

bubonicfred committed:
refactor: :heavy_minus_sign: switch some underscore calls to lodash

bubonicfred committed:
refactor: :heavy_minus_sign: switch some underscore calls to lodash

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 19f808a according to the output
from Prettier.

Details: https://github.com/bubonicfred/5minitz/pull/491

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 19f808a according to the output
from Prettier.

Details: https://github.com/bubonicfred/5minitz/pull/491

restyled-commits committed:
Restyled by clang-format

restyled-commits committed:
Restyled by clang-format

restyled-commits committed:
Restyled by prettier

restyled-commits committed:
Restyled by prettier

bubonicfred committed:
chore(package): :arrow_up: upgrade ldapjs, remove sinon-chai (unused)

bubonicfred committed:
chore(package): :arrow_up: Upgrade mongodb

deepsource-autofix[bot] committed:
refactor: replace with appropriate comparison operator

Avoid using `>= 0` and `< 0` when comparing container sizes.

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 6defce7 according to the output
from Prettier.

Details: https://github.com/bubonicfred/5minitz/pull/495

bubonicfred committed:
chore(package): :heavy_minus_sign: remove ddp

bubonicfred committed:
Bump chromedriver in tests

bubonicfred committed:
chore:

bubonicfred committed:
chore(package): :arrow_up: upgrade node-cron (#497)

bubonicfred committed:
Merge pull request #463 from bubonicfred/depfu/update/buildfix/npm/chromedriver-117.0.3

Update chromedriver → 117.0.3 (unknown)

bubonicfred committed:
chore: :heavy_minus_sign: Remove node-sass (deprecated

bubonicfred committed:
chore: :see_no_evil:

bubonicfred committed:
Merge pull request #498 from bubonicfred/buildfix-prune

Prune and update package.json

bubonicfred committed:
refactor: Use single import statement per module

bubonicfred committed:
Merge branch 'develop_new' into buildfix

bubonicfred committed:
docs(readme): Update Readme

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 04ceedf according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/499

bubonicfred committed:
chore: :see_no_evil: allow package-lock to be committed

bubonicfred committed:
Merge branch 'buildfix' of https://github.com/bubonicfred/5minitz into buildfix

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 3d5aaea according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/503

restyled-commits committed:
Restyled by clang-format

restyled-commits committed:
Restyled by prettier

bubonicfred committed:
Merge pull request #504 from bubonicfred/restyled/buildfix-imports

Restyle refactor: Use single import statement per module

bubonicfred committed:
Merge pull request #499 from bubonicfred/buildfix

Buildfix

depfu[bot] committed:
chore(package): update fs-extra to version 11.1.1

bubonicfred committed:
Merge pull request #503 from bubonicfred/buildfix-imports

refactor: Use single import statement per module

bubonicfred committed:
chore(package): :package: commit package-lock.json

bubonicfred committed:
update eslint config

bubonicfred committed:
Merge pull request #508 from bubonicfred/depfu/update/buildfix/npm/fs-extra-11.1.1

Update fs-extra → 11.1.1 (unknown)

bubonicfred committed:
Merge pull request #509 from bubonicfred/buildfix

Buildfix

bubonicfred committed:
Refactor (#510)

* refactor: :recycle:

* refactor

* style: format code with Prettier

This commit fixes the style issues introduced in 77a8345 according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/510

* Restyled by clang-format

* Restyled by prettier

* convert to dot notation

---------

Co-authored-by: deepsource-autofix[bot] <62050782+deepsource-autofix[bot]@users.noreply.github.com>
Co-authored-by: Restyled.io <commits@restyled.io>

depfu[bot] committed:
Update randomstring to version 1.3.0 (#515)

Co-authored-by: depfu[bot] <23717796+depfu[bot]@users.noreply.github.com>

bubonicfred committed:
Update issue templates (#514)

* Update issue templates

* style: format code with Prettier

This commit fixes the style issues introduced in abee01e according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/514

---------

Co-authored-by: deepsource-autofix[bot] <62050782+deepsource-autofix[bot]@users.noreply.github.com>

depfu[bot] committed:
Update fs-extra to version 11.1.1 (#516)

Co-authored-by: depfu[bot] <23717796+depfu[bot]@users.noreply.github.com>

bubonicfred committed:
Add eslint configs

bubonicfred committed:
chore(package):

bubonicfred committed:
Add precommit config

bubonicfred committed:
style:

bubonicfred committed:
refactor: use template literals instead of string concatenation

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in fb709cd according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/530

restyled-commits committed:
Restyled by clang-format

restyled-commits committed:
Restyled by prettier

bubonicfred committed:
refactor:

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in e45303c according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/532

bubonicfred committed:
refactor:

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 43c792b according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/533

bubonicfred committed:
refactor: convert to shorthands

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 502866a according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/534

restyled-commits committed:
Restyled by clang-format

restyled-commits committed:
Restyled by prettier

bubonicfred committed:
rm jquery-ui-touch-punch

bubonicfred committed:
refactor: switch to using lodash over underscore

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 5bc34bf according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/536

restyled-commits committed:
Restyled by clang-format

restyled-commits committed:
Restyled by prettier

bubonicfred committed:
feat: switch to using validation library for email checks

bubonicfred committed:
feat: migrate to reacti dict

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in fd4f3c0 according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/538

restyled-commits committed:
Restyled by clang-format

restyled-commits committed:
Restyled by prettier

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in ae7c39e according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/540

restyled-commits committed:
Restyled by clang-format

restyled-commits committed:
Restyled by prettier

bubonicfred committed:
inline multiple imports from same module

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in dbd8d18 according to the output
from Prettier.

Details: None

bubonicfred committed:
Replace reactive dict with session

bubonicfred committed:
refactor: :recycle:

bubonicfred committed:
Remove Session

bubonicfred committed:
refactor:

bubonicfred committed:
Replace Session with ReactiveDict

bubonicfred committed:
Replace Session with Reactive Dict + refactor

bubonicfred committed:
Reafctor

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 29746d1 according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/543

restyled-commits committed:
Restyled by clang-format

restyled-commits committed:
Restyled by prettier

bubonicfred committed:
test: Remove redundent tests

bubonicfred committed:
chore: Update Restyled config

bubonicfred committed:
refactor: Convert string concats to template literals

bubonicfred committed:
test: modify test to run oin v14

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 1c07b3b according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/547

bubonicfred committed:
Sessionnew (#545)

* Replace reactive dict with session

* refactor: :recycle:

* Remove Session

* refactor:

* Replace Session with ReactiveDict

* Replace Session with Reactive Dict + refactor

* Reafctor

* style: format code with Prettier

This commit fixes the style issues introduced in 29746d1 according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/543

* Replace Session with ReactiveDict

* refactor: Refactor and replace .set with .equals

* refactor

* style: format code with Prettier

This commit fixes the style issues introduced in 16b201d according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/545

* style: format code with Prettier

This commit fixes the style issues introduced in e2768a8 according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/545

* Remove duplicate import of ReactiveDict

* style: format code with Prettier

This commit fixes the style issues introduced in 981c8c8 according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/545

* refactor: Relocate openPrintDialog to before use

* style: format code with Prettier

This commit fixes the style issues introduced in 4efd2a3 according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/545

* Remove unsused variables

* Remove use of ==

* style: format code with Prettier

This commit fixes the style issues introduced in 53ee97b according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/545

* Replace Session with ReactiveDict

* style: format code with Prettier

This commit fixes the style issues introduced in 7bc1fca according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/545

* Remove unused variable

* style: format code with Prettier

This commit fixes the style issues introduced in 3b93030 according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/545

* style: format code with clang-format

* style: format code with prettier

---------

Co-authored-by: deepsource-autofix[bot] <62050782+deepsource-autofix[bot]@users.noreply.github.com>
Co-authored-by: Restyled.io <commits@restyled.io>

bubonicfred committed:
seperate faker use from random module

bubonicfred committed:
chore(package): move faker to dev dependencies

bubonicfred committed:
update and prune imports

bubonicfred committed:
change import

restyled-io[bot] committed:
Restyle Random (#550)

* style: format code with clang-format

* style: format code with prettier

---------

Co-authored-by: Restyled.io <commits@restyled.io>

bubonicfred committed:
Convert square to dot notation.

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in cc99219 according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/551

bubonicfred committed:
Replace Session with Reactive Dict

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 4d2f63a according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/551

bubonicfred committed:
simplify import

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 8d134a1 according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/551

bubonicfred committed:
add await to async

bubonicfred committed:
Revert "add await to async"

This reverts commit 554afbee4524430d4c9d3dbbc33bff0625949eb4.

restyled-commits committed:
style: format code with clang-format

restyled-commits committed:
style: format code with prettier

bubonicfred committed:
Refactors

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in e6698ce according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/553

restyled-commits committed:
style: format code with clang-format

restyled-commits committed:
style: format code with prettier

bubonicfred committed:
feat: :goal_net: add error objects to rejects

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 80eef82 according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/553

bubonicfred committed:
feat: :goal_net: add error object missed in previous commit

bubonicfred committed:
chore: add licensebat config

bubonicfred committed:
refactor: convert funtion to arrow function

bubonicfred committed:
Add early return

bubonicfred committed:
refactor:

bubonicfred committed:
style: switch to lodash +style

bubonicfred committed:
style: Style HTML + Less

bubonicfred committed:
refactor:

bubonicfred committed:
Revert "style: Style HTML + Less"

This reverts commit e2c42727422140ed8ec2fb76451135fe0f941e0f.

bubonicfred committed:
Refactor

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in b955b17 according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/559

restyled-commits committed:
style: format code with clang-format

restyled-commits committed:
style: format code with prettier

bubonicfred committed:
Refactor

bubonicfred committed:
Merge branch 'Refactornew' of https://github.com/bubonicfred/4minitz into Refactornew

bubonicfred committed:
Refactor

bubonicfred committed:
refactor meetingSeries

bubonicfred committed:
refactor topic*

bubonicfred committed:
Refactor expImp

bubonicfred committed:
Refactor helpers

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 33de32f according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/561

restyled-commits committed:
style: format code with clang-format

restyled-commits committed:
style: format code with prettier

bubonicfred committed:
Refactor Services

bubonicfred committed:
Merge branch 'Refactornew' of https://github.com/bubonicfred/4minitz into Refactornew

bubonicfred committed:
Refactor Collections

bubonicfred committed:
Refactor globals

bubonicfred committed:
refactor server

bubonicfred committed:
Refactor templates

bubonicfred committed:
Refactor *

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/4minitz/4minitz?pullRequest=589) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/4minitz/4minitz?pullRequest=589) before we can accept your contribution.<br/>**0** out of **4** committers have signed the CLA.<br/><br/>:x: restyled-io[bot]<br/>:x: bubonicfred<br/>:x: deepsource-autofix[bot]<br/>:x: restyled-commits<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/4minitz/4minitz?pullRequest=589) it.</sub>

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 999c6bd according to the output
from Prettier.

Details: https://github.com/bubonicfred/4minitz/pull/563

bubonicfred committed:
Replace Session with ReactiveDict

bubonicfred committed:
Remove redundant return
